### ISSUE TITLE
SNES adding games issue. Wierd Shell Error * edit. Have Work around*

### ISSUE THREAD

**Issue by Bluebpy** [ClusterM/hakchi2#760]
Hello,

Im using Hakchi V2.21c

I flashed the snes classic rom and everything is good. When I try to add games I get this error.

I tried using different usb ports. Using USB 2.0 to be safe. Tried to flash back to original and try again. Nothing seems to work and this error happens every time. 

Help please?
Screenshot is here
![snes issue](https://user-images.githubusercontent.com/32773055/31550985-277ab676-b001-11e7-8299-2dca4a22cff6.PNG)

**Comment by doomguy255** [comment]
Yeah I'm also getting it.
All I'm did was the following.

1. Dump original kernel.
2. Load a custom kernel.
3. Add games and sync.

Results: I get that same shell command error.

**Comment by Bluebpy** [comment]
Yeah i tried different ways but around 90% progress I get the error. The snes boots but has only original games. Not sure what is causing this.

**Comment by DecanFrost** [comment]
I posted this problem . But I think I found a solution ..  ot the best but seems to work.  I redownloaded the app. Tossed it in a new folder and then added the games folder and configure folder. To it and it worked no problem.  I tried to just overright the files back to the original folder buy got the error again. So I think somehow the folder gets corrupt or a sub folder did

**Comment by DecanFrost** [comment]
Forgot to mention.  Flash stock kernel with new install and then reflash custom

**Comment by Bluebpy** [comment]
Ok I came up with a fix.

Not sure what the cause is but here it is.

So I had my original NES classic Hakchi folder and the snes was in the same directory when I downloaded this new build. To fix this issue download the build and install it separate in another directory and do not mix and match NES and SNES classic directory or files. 

Once I did that the issue went away and everything is working fine.
